### PR TITLE
Solana FAQ Editorial added to Sidebar

### PR DESCRIPTION
**Pull request by samgutentag**

### CODE REVIEW

File: docs/solana/migration-faq.mdx [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+pagination_label: Migration FAQ
+sidebar_label: Migration FAQ
+description: Helium Documentation
+image: https://docs.helium.com/img/link-image.png
+slug: /solana/migration-faq
+---
+
+# FAQ - Solana Migration
+
+## Why Does My Helium Wallet App Crash When I Open It?
```
Comment (jthiller) [inline docs/solana/migration-faq.mdx]: Let's move this one way down the list. A bit nicer to kick off with the SOL question. Odds are this is quickly solved as long as people are getting automatic updates.

**Review by jthiller — CHANGES_REQUESTED** [review]
Do we need a netlify redirect for the changed /faq/solana-migration file?

File: docs/solana/migration-faq.mdx (hunk 1 above)
Comment (samgutentag) [reply docs/solana/migration-faq.mdx]: fixed!

**Comment by samgutentag** [comment]
> Do we need a netlify redirect for the changed /faq/solana-migration file?

the old page name was never actually linked on the sidebar, no need for a redirect

**Review by jthiller — APPROVED** [review]

**Pull request merged**